### DISCUSSION TITLE
FLOWUnsteady failed to compile, has property of NULL project

### DISCUSSION THREAD

**Discussion by Turbulence1116** [byuflowlab/FLOWUnsteady#87] [Q&A]
![disaster1](https://github.com/byuflowlab/FLOWUnsteady/assets/141818598/6cb93e9b-e8d5-4275-ab95-ba2124fa4158)
![disaster2](https://github.com/byuflowlab/FLOWUnsteady/assets/141818598/0eb80c00-ffc3-4324-bc24-733e4a964a49)
![disaster3](https://github.com/byuflowlab/FLOWUnsteady/assets/141818598/3dd40ef5-1be2-455e-9fd5-9ee04b914773)
![disaster4](https://github.com/byuflowlab/FLOWUnsteady/assets/141818598/f6549ab0-e513-488a-add9-66b2d83d3f0b)

Dear all,
I just had disaster when I was trying to run XFOIL in julia, I add XFOIL first (picture 1), then the error message says that PyPlot not found in current path (picture 2), after I added (It was actually updated to newer version) PyPlot, the FLOWUnsteady and FLOWNoise cannot be pre-compiled (picture 2,3). 
I tried to re-install all the packages and julia, the issue is still there. Then I downgraded the PyPlot version to previous one, the situation seems to get worse (picture 4). 
Something must be wrong with my own python environment and so far I have no clue of it and how it was caused. Any hint on this will be very appreciated!

**Comment by Turbulence1116** [comment]
actually problem was solved, the PyPlot version 2.11.1 works now.

**Comment by EdoAlvarezR** [comment]
Related to #88 , thanks for pointing this out!
